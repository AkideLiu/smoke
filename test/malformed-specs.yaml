tests:
  - name: malformed-specs
    args:
      - fixtures/malformed-specs
    exit-status: 1
    stdout:
      file: io/malformed-specs.out
