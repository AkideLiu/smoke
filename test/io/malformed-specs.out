empty
  The test specification "fixtures/malformed-specs/empty.yaml" is invalid:
    Aeson exception:
    Error in $: expected Suite, encountered Null
invalid-yaml
  The test specification "fixtures/malformed-specs/invalid-yaml.yaml" is invalid:
    YAML parse exception at line 3, column 0,
    while parsing a flow node:
    did not find expected node content
test-as-dict
  The test specification "fixtures/malformed-specs/test-as-dict.yaml" is invalid:
    Aeson exception:
    Error in $.tests: expected [a], encountered Object

3 tests, 3 failures
